internal/provider/test_constants.go

```diff
@@ -88,3 +87,50 @@ resource "aci_l3out_loopback_interface_profile" "test" {
   addr           = "1.2.3.5"
 }
 `
+
+const testConfigDefaultMin = ``
+
+const testConfigFvTenantMinDependencyWithPkiTP = testConfigFvTenantMin + `
```

Comment:
why are these defined as constants? and not generated?

Reply:
Removed